<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<!-- saved from url=(0016)http://localhost -->
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<style type="text/css" id="internalStyle" xmlns="">
		body {
		  font-family: Verdana, Gill, Helvetica, Sans-serif ;
		  font-size: 0.8em;
		  font-weight: 500;
		  color: #000020;
		  background-color: #FFFFFF;
		}
		div.itemDescription {
		  margin-right: 2em;
		  margin-bottom: 2em;
		}
		h1 {
		  font-size: 1.5em;
		  margin-top: 0;
		  margin-bottom: 5px;
		}
		h1.idHeading {
		  color: #008FAF;
		  text-align: center;
		}
		h1.gpHeading {
		  color: black;
		}
		span.idHeading {
		  color: #007799;
		  font-weight: bold;
		}
		.center {
		  text-align: center;
		  margin-top: 5px;
		  margin-bottom: 5px;
		}
		img {
		  width: 210px;
		  border-width: 1px;
		  border-style: outset;
		}
		img.center {
		  text-align: center;
		  display: block;
		  border-color: #666666;
		}
		img.enclosed {
		  width: 60%;
		}
		img.gp {
		  width: auto;
		  border-style: none;
		  margin-top: -1.2em;
		}
		.noThumbnail {
		  color: #888888;
		  font-size: 1.2em;
		  border-width: 1px;
		  border-style: solid;
		  border-color: black;
		  padding: 3em 3em;
		  position: relative;
		  text-align: center;
		  width: 210px;
		  height: 140px;
		}
		.noContent {
		  color: #888888;
		}
		.itemInfo p {
		  margin-top: -0.1em;
		}
		.itemInfo img {
		  width: auto;
		  border: none;
		}
		.gpItemInfo p {
		  margin-top: -1.2em;
		}
		div.box {
		  margin-left: 1em;
		}
		div.hide {
		  display: none;
		}
		div.show {
		  display: block;
		}
		span.hide {
		  display: none;
		}
		span.show {
		  display: inline-block;
		}
		.backToTop a {
		  color: #DDDDDD;
		  font-style: italic;
		  font-size: 0.85em;
		}
		h2 {
		  font-size: 1.2em;
		}
		h2.gp {
		  color: #00709C;
		}
		.gpsubtitle {
		  color: black;
		  font-size: 1.2em;
		  font-weight: normal;
		}
		.gptags {
		  color: black;
		  font-size: 0.8em;
		  font-weight: normal;
		}
		.head {
		  font-size: 1.3em;
		}
		a:link {
		  color: #098EA6;
		  font-weight: normal;
		  text-decoration: none;
		}
		a:visited {
		  color: #098EA6;
		  text-decoration: none;
		}
		a:link:hover, a:visited:hover {
		  color: #007799;
		  background-color: #C6E6EF;
		}
		h2.iso a {
		  color: #007799;
		  font-weight: bold;
		}
		.iso a:link {
		  color: #007799;
		  text-decoration: none;
		}
		.iso a:visited {
		  color: #007799;
		  text-decoration: none;
		}
		.iso a:link:hover, .iso a:visited:hover {
		  color: #006688;
		  background-color: #C6E6EF;
		}
		h2.fgdc a {
		  color: #888888;
		  font-weight: bold;
		}
		.fgdc a:link {
		  color: #888888;
		  text-decoration: none;
		}
		.fgdc a:visited {
		  color: #888888;
		  text-decoration: none;
		}
		.fgdc a:link:hover, .fgdc a:visited:hover {
		  color: #777777;
		  background-color: #C6E6EF;
		}
		h3 {
			font-size: 1em; 
			color: #00709C;
		}
		.backToTop {
		  color: #AAAAAA;
		  margin-left: 1em;
		}
		p.gp {
		  margin-top: .6em;
		  margin-bottom: .6em;
		}
		ul ul {
		  list-style-type: square;
		}
		ul li.iso19139heading {
		  margin-left: -3em;
		  list-style: none;
		  font-weight: bold;
		  color: #666666;
		}
		dl {
		  margin: 0;
		  padding: 0;
		}
		dl.iso {
		  background-color: #F2F9FF;
		}
		dl.esri {
		  background-color: #F2FFF9;
		}
		dl.subtype {
		  width: 40em;
		  margin-top: 0.5em;
		  margin-bottom: 0.5em;
		  padding: 0;
		}
		dt {
		  margin-left: 0.6em;
		  padding-left: 0.6em;
		  clear: left;
		}
		.subtype dt {
		  width: 60%;
		  float: left;
		  margin: 0;
		  padding: 0.5em 0.5em 0 0.75em;
		  border-top: 1px solid #006400;
		  clear: none;
		}
		.subtype dt.header {
		  padding: 0.5em 0.5em 0.5em 0;
		  border-top: none;
		}
		dd {
		  margin-left: 0.6em;
		  padding-left: 0.6em;
		  clear: left;
		}
		.subtype dd {
		  float: left;
		  width: 25%;
		  margin: 0;
		  padding: 0.5em 0.5em 0 0.75em;
		  border-top: 1px solid #006400;
		  clear: none;
		}
		.subtype dd.header {
		  padding: 0.5em 0.5em 0.5em 0;
		  border-top: none;
		}
		.isoElement {
		  font-variant: small-caps;
		  font-size: 0.9em;
		  font-weight: normal;
		  color: #006688;
		}
		.esriElement {
		  font-variant: small-caps;
		  font-size: 0.9em;
		  font-weight: normal;
		  color: #006688;
		}
		.element {
		  font-variant: small-caps;
		  font-size: 0.9em;
		  font-weight: normal;
		  color: #666666;
		}
		unknownElement {
		  font-variant: small-caps;
		  font-size: 0.9em;
		  font-weight: normal;
		  color: #333333;
		}
		.sync {
		  color: #006400;
		  font-weight: bold;
		  font-size: 0.9em;
		}
		.syncOld {
		  color: #888888;
		  font-weight: bold;
		  font-size: 0.9em;
		}
		.textOld {
		  color: #999999;
		}
		.code {
		  font-family: monospace;
		}
		pre.wrap {
		  width: 96%;
		  font-family: Verdana, Gill, Helvetica, Sans-serif ;
		  font-size: 1em;
		  margin: 0 0 1em 0.6em;
		  white-space: pre-wrap;       /* css-3 */
		  white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
		  white-space: -pre-wrap;      /* Opera 4-6 */
		  white-space: -o-pre-wrap;    /* Opera 7 */
		  word-wrap: break-word;       /* Internet Explorer 5.5+ */
		}
		pre.gp {
		  font-family: Courier New, Courier, monospace;
		  line-height: 1.2em;
		}
		.gpcode {
		  margin-left:15px;
		  border: 1px dashed #ACC6D8;
		  padding: 10px;
		  background-color:#EEEEEE;
		  height: auto;
		  overflow: scroll; 
		  width: 96%;
		}
		tr {
		  vertical-align: top;
		}
		th {
		  text-align: left;
		  background: #dddddd;
		  vertical-align: bottom;
		  font-size: 0.8em;
		}
		td {
		  background: #EEEEEE;
		  color: black;
		  vertical-align: top;
		  font-size: 0.8em;
		}
		td.description {
		  background: white;
		}
	  </style>
</head>
<body oncontextmenu="return true">
<h1 class="gpHeading" xmlns="">Import Data</h1>
<h2 class="gp" xmlns="">Titleâ€ƒ
		<span class="gpsubtitle">Import Data</span>
</h2>
<h2 class="gp" xmlns="">Summary</h2>
<div class="gpItemInfo" xmlns="">
<p><DIV STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This script tool: (1) creates a File Geodatabase if one does not exist, (2) imports spatially referenced genetic data from a simple flat file format (SRGD.csv) to a new Feature Class located in the File Geodatabase, and (3) creates a copy of the input data in a table located in the File Geodatabase.</SPAN></P><P><SPAN STYLE="font-weight:bold;">Required Inputs:</SPAN></P><UL><LI><P><SPAN>Spatially referenced genetic data formatted according to the SRGD.csv file format. Data can be for identified individuals or for genetic samples. </SPAN></P><P><SPAN>Data will be automatically sorted into the categories Genetic, Identification, Location and Other. The user should look to make sure the appropriate fields are categorized correctly. If changes are needed, the user can un-check and/or re-check data fields where appropriate.</SPAN></P><P><SPAN /></P><P><SPAN>NOTE: The SRGD.csv format includes the </SPAN><SPAN STYLE="font-style:italic;">minimum </SPAN><SPAN>data requirements for basic </SPAN><SPAN STYLE="font-style:italic;">geneGIS </SPAN><SPAN>functionality. Additional data can be included.</SPAN></P></LI></UL><P><SPAN STYLE="font-weight:bold;">Optional Inputs:</SPAN></P><UL><LI><P><SPAN>If known, a spatial reference for the point data (recommended).</SPAN></P><P><SPAN /></P><P><SPAN>NOTE: The default spatial reference is Geographic WGS 1984.</SPAN></P></LI></UL><P><SPAN STYLE="font-weight:bold;">Outputs:</SPAN></P><UL><LI><P><SPAN>A new file geodatabase (if one does not already exist).</SPAN></P></LI><LI><P><SPAN>A new feature class located within the file geodatabase.</SPAN></P></LI><LI><P><SPAN>A copy of the data in a table located within the file geodatabase.</SPAN></P></LI></UL><P /></DIV></DIV></DIV></p><br />
</div>
<h2 class="gp" xmlns="">Usage</h2>
<div class="gpItemInfo" xmlns="">
<p><DIV STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This script tool can be used to import spatially referenced genetic data into a feature class that is viewable in ArcGIS.</SPAN></P></DIV></DIV></DIV></p><br />
</div>
<h2 class="gp" xmlns="">Syntax</h2>
<div xmlns="">
<p style="margin-top: -1.2em">ClassifiedImport (SRGD_Input_File, {Spatial_Reference}, File_Geodatabase_Location, File_Geodatabase_Name, Genetic_Columns, Identification_Columns, Location_Columns, {Other_Columns}) <br /><br />
</p>
<table width="100%" border="0" cellpadding="5">
<tbody>
<tr>
<th width="30%">
<b>Parameter</b>
</th>
<th width="50%">
<b>Explanation</b>
</th>
<th width="20%">
<b>Data Type</b>
</th>
</tr>
<tr>
<td class="info">SRGD_Input_File</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>The
        following data fields for the SRGD input file MUST be
        formatted as
        follows:</SPAN></P><UL><LI><P><SPAN
        STYLE="font-weight:bold;">Sample_ID
        </SPAN><SPAN>(required)</SPAN></P><P><SPAN>A
        unique identifier for each data record or encounter with an
        animal that has an associated spatial (latitude, longitude)
        and temporal element (date, time) resulting in the
        collection of a sample (</SPAN><SPAN
        STYLE="font-style:italic;">e.g.</SPAN><SPAN>,
        biopsy, skin, ear tag) and/or a photograph, tag or band
        that will allow the same individual to be resighted or
        recaptured. This information may have been used to
        determine an individualÃ¢â‚¬â„¢s
        identity.</SPAN></P></LI></UL><UL><LI><P><SPAN
        STYLE="font-weight:bold;">Individual_ID</SPAN></P><P><SPAN>A
        unique identifier for each known individual in the dataset.
        This information may have been derived from some previous
        type of analysis (</SPAN><SPAN
        STYLE="font-style:italic;">e.g.</SPAN><SPAN>,
        determination of identity using genotyping or
        photo-identification methods).
        </SPAN></P></LI></UL><UL><LI><P><SPAN
        STYLE="font-weight:bold;">Latitude
        </SPAN><SPAN>(required)</SPAN></P><P><SPAN>The
        latitude, in decimal degrees, where the sample or
        photograph was collected. Latitudes are positive north of
        the equator and negative south of the
        equator.</SPAN></P></LI></UL><UL><LI><P><SPAN
        STYLE="font-weight:bold;">Longitude
        </SPAN><SPAN>(required)</SPAN></P><P><SPAN>The
        longitude, in decimal degrees, where the sample or
        photograph was collected. Longitudes are positive east of
        the Prime Meridian and negative west of the Prime
        Meridian.</SPAN></P><P><SPAN
        /></P><P><SPAN>NOTE: the default
        spatial reference, unless indicated by the user during data
        import, is Geographic
        (WGS84).</SPAN></P><P><SPAN
        /></P><P><SPAN>Alternatively, data can
        specify X and Y columns for projected datasets, such as
        coordinates stored in
        UTM.</SPAN></P><P><SPAN
        /></P></LI><LI><P><SPAN
        STYLE="font-weight:bold;">Date_Time</SPAN><SPAN>(Date
        required; Time
        optional)</SPAN></P><P><SPAN>The
        date and time for when the sample or photograph was
        collected. We recommend ISO 8601 date format (YYYY/MM/DD
        hh:mm:ss).</SPAN></P><P
        /></LI></UL><UL><LI><P><SPAN
        STYLE="font-weight:bold;">Region</SPAN></P><P><SPAN>The
        general region from where the sample or photograph was
        collected. This can be based on political or geographic
        boundaries (</SPAN><SPAN
        STYLE="font-style:italic;">e.g.</SPAN><SPAN>,
        oceans, countries, states, etc.) or the study design
        (</SPAN><SPAN
        STYLE="font-style:italic;">e.g.</SPAN><SPAN>,
        a specific quadrat or
        plot).</SPAN></P><P><SPAN
        /></P></LI><LI><P><SPAN
        STYLE="font-weight:bold;">Sex
        </SPAN><SPAN>(*)</SPAN></P><P><SPAN>The
        genetic sex (</SPAN><SPAN
        STYLE="font-style:italic;">e.g.</SPAN><SPAN>,
        M, F, U) of the sample or individual, if
        known.</SPAN></P><P><SPAN
        /></P></LI><LI><P><SPAN
        STYLE="font-weight:bold;">Haplotype</SPAN><SPAN>(*)</SPAN></P><P><SPAN>The
        mitochondrial haplotype of the sample or individual, if
        known.</SPAN></P><P><SPAN
        /></P></LI><LI><P><SPAN
        STYLE="font-weight:bold;">L_YourLocusName</SPAN><SPAN>(*)</SPAN></P><P><SPAN>The
        name of a specific locus. For biallelic data
        (microsatellites, SNPs), there should be 2 columns with the
        same locus name. Each column should hold a value for each
        of the alleles for that
        locus.</SPAN></P><P><SPAN>Include
        as many loci and allelic values as necessary for your
        data.</SPAN></P><P><SPAN>The L_
        prefix will allow the Import Tool to recognize correctly
        the loci.
        </SPAN></P><P><SPAN>ArcGIS does not
        allow more than one field with the same name. Upon import
        into ArcGIS, the name for each locus will be appended as
        follows: L_YourLocusName_1, L_YourLocusName_2.
        </SPAN><SPAN /></P><P><SPAN>*
        </SPAN><SPAN><SPAN>NOTE: These fields are
        considered optional because any combination of Sex,
        Haplotype and Loci data are possible. However,
        </SPAN></SPAN><SPAN
        STYLE="font-style:italic;font-weight:bold;">at least one
        is required</SPAN><SPAN><SPAN>to make use
        of the intended </SPAN></SPAN><SPAN
        STYLE="font-style:italic;"><SPAN>geneGIS
        </SPAN></SPAN><SPAN>functionality.</SPAN></P><P><SPAN
        /></P></LI></UL><P><SPAN>Additional
        fields can be included as
        needed.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">File</td>
</tr>
<tr>
<td class="info">Spatial_Reference (Optional) </td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>The
        default spatial reference is Geographic WGS 1984.
        </SPAN></P><P><SPAN>If data were
        collected with another spatial format, specify it here.
        </SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Spatial Reference</td>
</tr>
<tr>
<td class="info">File_Geodatabase_Location</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>The
        location where the new file geodatabase will be placed or
        the location of an already existing file
        geodatabase.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Folder</td>
</tr>
<tr>
<td class="info">File_Geodatabase_Name</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>The
        name of the new or already existing file
        geodatabase.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">String</td>
</tr>
<tr>
<td class="info">Genetic_Columns</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This
        section should include any relevant genetic data (Sex,
        Haplotype, Loci) within the dataset being imported. The
        user should ensure all appropriate fields are checked
        and/or un-checked.
        </SPAN></P><P><SPAN>For loci data,
        ArcGIS does not allow more than one field with the same
        name. Upon import into AcGIS, the name for each locus will
        be appended as follows: L_YourLocusName_1,
        L_YourLocusName_2.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Multiple Value</td>
</tr>
<tr>
<td class="info">Identification_Columns</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This
        section should include any relevant indentification data
        (Sample_ID, Individual_ID) within the dataset being
        imported. The user should ensure all appropriate fields are
        checked and/or
        un-checked.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Multiple Value</td>
</tr>
<tr>
<td class="info">Location_Columns</td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This
        section should include the latitude and longitude data for
        the dataset being imported or X and Y columns for projected
        data.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Multiple Value</td>
</tr>
<tr>
<td class="info">Other_Columns (Optional) </td>
<td class="info" align="left">
<span style="font-weight: bold">Dialog Reference</span><br /><DIV
        STYLE="text-align:Left;"><DIV><DIV><P><SPAN>This
        section can include any other relevant data that the user
        wants to include in the import by either checking and/or
        un-checking appropriate
        fields.</SPAN></P></DIV></DIV></DIV><p><span class="noContent">There is no python reference for this parameter.</span></p></td>
<td class="info" align="left">Multiple Value</td>
</tr>
</tbody>
</table>
</div>
<h2 class="gp" xmlns="">Code Samples</h2>
<div class="gpItemInfo" xmlns="">
<p><span class="noContent">There are no code samples for this tool.</span></p>
</div>
<h2 class="gp" xmlns="">Tags</h2>
<div class="gpItemInfo" xmlns="">
<p>import spatially explicit genetic data, creates file geodatabase, creates feature class</p>
</div>
<h2 class="gp" xmlns="">Credits</h2>
<div class="gpItemInfo" xmlns="">
<p>
<p>Created by the geneGIS team: Dori Dick, Shaun Walbridge, Scott Baker, Dawn Wright</p>
</p><br />
</div>
<h2 class="gp" xmlns="">Use limitations</h2>
<div class="gpItemInfo" xmlns="">
<p><span class="noContent">There are no use limitations for this item.</span></p>
</div>
</body>
</html>
